CRM/Financial/BAO/ExportFormat/CSV.php

```diff
@@ -66,9 +68,6 @@ public function export($exportParams) {
       $this->$func();
     }
 
-    // now do general journal entries
-    $this->exportTRANS();
```

Comment:
If you look at the exportTRANS() function defined just below, it's empty. I assume there is no point in calling an empty function?

This probably crept in while copy pasting from its neighbor IIF.php.